src/media-gallery/media-gallery.js

```diff
@@ -123,16 +143,34 @@ component.template = `
             <span class="media-gallery__screen-reader-only">Remove</span>
         </button>
     </div>
+    <div class="media-list__media-container" ng-if="vm.view === vm.galleryView.LIST"
+         ng-repeat="media in vm.media" track by media.fileName>
```

Comment:
misplaced endquote

Reply:
```
ng-repeat="media in vm.media" track by media.fileName
```
should be 
```
ng-repeat="media in vm.media track by media.fileName"
```

Reply:
Oops I fixed the one on the line below thinking that's what you meant. Got it.